src/main/java/binnie/core/craftgui/database/PageSpeciesOverview.java

```diff
@@ -49,7 +49,7 @@ public void onValueChanged(IAlleleSpecies species) {
 		String descBody = EnumChatFormatting.ITALIC.toString();
 		String descSig = "";
 
-		if (desc == null || desc.isEmpty()) {
+		if (desc.contains(".species.") || desc.contains("for.description.")) {
```

Comment:
Yes. `desc` not null and not empty.
This is contains localized string or unlocalized key (if localization not exists).
I think, in Forestry 3.6 `desc` was null or empty.
I will check this again when I translate databases.